spec/mailers/track_mailer_spec.rb

```diff
@@ -28,7 +28,7 @@
                            :url_name => 'test-name',
                            :get_locale => 'en',
                            :should_be_emailed? => true)
-        klass = ActiveRecord::Relation::ActiveRecord_Relation_User
+        klass = ActiveRecord::Relation
```

Comment:
What was the failure? How did you come to figure out that removing `::ActiveRecord_Relation_User` fixed it?